core/base/ConfigManager.py

```diff
@@ -227,11 +230,12 @@ def getSnipsConfiguration(self, parent: str, key: str, createIfNotExist: bool =
 		:param createIfNotExist: If that conf doesn't exist, create it
 		:return: config value
 		"""
-		if createIfNotExist:
-			self._snipsConfigurations[parent] = self._snipsConfigurations.get(parent, dict())
-			self._snipsConfigurations[parent][key] = self._snipsConfigurations[parent].get(key, '')
+		if createIfNotExist and not key in self._snipsConfigurations[parent]:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [test for membership should be 'not in'](https://app.codacy.com/app/ProjectAlice/ProjectAlice/pullRequest?prid=4518670)